frontend/libs/studio-components/src/style/studioBetaTag.module.css

```
@@ -0,0 +1,9 @@
+.isBeta:after {
```

`after` er et pseudo-element, ikke en pseudo-klasse, så det må være to kolon, selv om enkelte nettlesere aksepterer at det kun er ett.
```suggestion
.isBeta::after {
```

---

Fixed!